crypto/bindings/conversion-verifier-index.ts

```diff
@@ -115,6 +115,12 @@ function verifierIndexConversionPerField(
     let mulComm = core.polyCommToRust(evals[6]);
     let emulComm = core.polyCommToRust(evals[7]);
     let endomulScalarComm = core.polyCommToRust(evals[8]);
+    let xorComm = MlOption.mapFrom(evals[9], core.polyCommToRust);
```

Comment:
xorComm and rangeCheck0Comm are the same

Reply:
fixed in subsequent commit